### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

**Comment by bricelam** [comment]
Oops, bad title. I didn't mean to imply that non-en-US spellings were typos. :blush:

**Comment by anpete** [comment]
:shipit:

File: test/Microsoft.EntityFrameworkCore.Tests/Utilities/MultigraphTest.cs [hunk 1]
```diff
@@ -202,7 +202,7 @@ public void AddEdges_throws_on_verticies_not_in_the_graph()
         }
 
         [Fact]
-        public void AddEdge_updates_incomming_and_outgoing_neighbours()
+        public void AddEdge_updates_incomming_and_outgoing_neighbors()
```
Comment (IDisposable) [inline test/Microsoft.EntityFrameworkCore.Tests/Utilities/MultigraphTest.cs]: Too many `m`

`AddEdge_updates_incoming_and_outgoing_neighbors`

**Comment by bricelam** [comment]
I reverted the changes to internal APIs and tests.

**Pull request merged**